package.json

```diff
@@ -53,9 +53,10 @@
     "@types/http-errors": "^1.6.3",
     "@types/morgan": "^1.9.2",
     "@types/triple-beam": "^1.3.2",
+    "eslint": "^7.24.0",
+    "jest": "^26.6.3",
```

Comment:
These can be removed now, right?

Reply:
Yup, exactly!

Reply:
Once we make a new release and update `@substrate/dev` I will also adjust the scripts for `lint` and `test` etc.